packages/devtools_app/lib/src/flutter/table.dart

```diff
@@ -412,6 +422,67 @@ class TreeTableState<T extends TreeNode<T>> extends State<TreeTable<T>>
       ..forEach(_sort);
   }
 
+  void _handleKeyEvent(RawKeyEvent event, ScrollController scrollController,
+      BoxConstraints constraints) {
+    if (event is! RawKeyDownEvent) return;
+
+    // If there is no selected node, choose the first one.
+    if (selectedNode == null) {
```

Comment:
we should only make a selection if one of the arrow keys are hit. As is, this will select a node if any key is hit.

Reply:
done